spec/features/admin/admin_views_all_users_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 feature "admin views all users" do
   scenario "successfully" do
+    skip
```

Comment:
Consider not adding this spec file because it is not in scope for this PR

Reply:
Gone in sha: 07105f5